src/lib/utils/createSendEventForHits.ts

```diff
@@ -60,7 +60,8 @@ const buildPayload: BuildPayload = ({
       return null;
     }
   }
-  const hitsArray = Array.isArray(hits) ? hits : [hits];
+  // `Array.from(hits)` is to remove `hits.__escaped`
+  const hitsArray = Array.isArray(hits) ? Array.from(hits) : [hits];
```

Comment:
Array.from isn't compatible with IE

Reply:
ah right. https://github.com/algolia/instantsearch.js/pull/4667/commits/d3868b519395a21ca95d9b239206d8265ba0669e

Reply:
you should instead remove the property if you don't want it to show up, or map over it

Reply:
https://github.com/algolia/instantsearch.js/pull/4667/commits/3b541f54c5ea3d8a41f6f5e9b3fac18a6a7f8530 WDTY?